pom.xml

```diff
@@ -178,6 +178,7 @@
         <graphql-spqr.version>0.0.4</graphql-spqr.version>
         <gridgain.version>6.6.9</gridgain.version>
         <guava.version>28.1-jre</guava.version>
+        <reactor.version>0.1.0</reactor.version>
```

Comment:
Please rename to `xe-rates-client.version` and also try to order the properties alphabetically.

Reply:
Fixed